docs/Makefile

```diff
@@ -20,11 +20,11 @@ docset: html
 	convert $(SPHINXPROJ).docset/icon@2x.png -resize 16x16 $(SPHINXPROJ).docset/icon.png
 
 rebuild:
-	rm -rf source/generated && make clean && make html
+	rm -rf source/generated && find ../ignite -name *.pyc -delete && rm -rf source/__pycache__ && make clean && make html
```

Comment:
Honestly, I'm not sure if this is really needed. If you want this command, then introduce another command, e.g. `clean_cache_rebuild`